docs/can-view-import.md

```diff
@@ -16,6 +16,30 @@ Statically import a module from within a [can-stache] template. *MODULE_NAME* wi
 
 @param {moduleName} [MODULE_NAME] A module that this template depends on.
 
+@signature `<can-import from="MODULE_NAME" module.default:to="VAR_NAME" />`
+
+Statically import a module from within a [can-stache] template. *MODULE_NAME* will be imported before the template renders. *VAR_NAME* will be set during initial template rendering.
+
+```
+<can-import from="app/helpers/properCase" module.properCase:to="properCase" />
```

Comment:
It might be helpful to use different variable names so people don't think the modules export has to match the name of the property you're creating on the let context.